lib/src/progressBar/progressTheme.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { ProggressTheme } from "./progressBar.component";
+
+@Pipe({ name: "proggressTheme" })
+export class ProgressThemePipe implements PipeTransform {
+    transform(value: ProggressTheme) {
+        switch (value) {
+            case "danger":
+                return "bg-danger";
+            case "warning":
+                return "bg-warning";
+            case "info":
+                return "bg-info";
+            default:
+                return "bg-success";
+        }
+    }
```

Comment:
why not just `return "bg-" + value`  ?

Reply:
I think that will defeat the purpose of having  progressTheme type, perhaps we can replace it with string then. Also imagine a user passing an invalid type , since there is no default, then it will match neither of the classes.

Reply:
I think that would be their fault, and it's easier to capture a mistake when you can see the glitch afterwards. I think you can use `bg-${value}`

Reply:
There is no mistake here though. The result is a theme , displaying a totally different. I will know right away if i am using the wrong one. Plus regardless whether user pass the right or wrong one, there must always be a theme, a default one at that.

Reply:
What if someone wants to use it with a custom class like `bg-nuru`? the switch will not allow that.

Reply:
Why should it allow it when it doesn't recognise it as a theme and doesn't have a definition for it ? All the themes there are predefined and if you need another, i think you should classNAmes.